manifests/vm.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubevirt.io/v1alpha2
+apiVersion: kubevirt.io/v1alpha3
 kind: VirtualMachine
 metadata:
   name: testvm
```

Comment:
We need to removal volumeNames from here as well.

Reply:
Ah yes!